packages/amazonq/src/lsp/chat/messages.ts

```diff
@@ -258,3 +363,16 @@ async function handleCompleteResult<T>(
     })
     disposable.dispose()
 }
+
+async function handleRequest(
```

Comment:
nit: Maybe this is intuitive to people familiar with the file. But should this be renamed to `handleChatRequest()` since there is also a different "request" in the language server context (eg `langaugeClient.sendRequest`)